RegularWebApp/views/index.jade

```diff
@@ -37,11 +36,34 @@ block script
     }
 
     function loginEmailCode() {
-      var lock = new Auth0LockPasswordless('#{env.AUTH0_CLIENT_ID}', '#{env.AUTH0_DOMAIN}');
-      lock.emailcode( {callbackURL: '#{env.AUTH0_CALLBACK_URL}'} );
+      var opts =  {
+        allowedConnections: ['email'],
+        auth: {
+          redirectUrl: '#{env.AUTH0_CALLBACK_URL}',
+          responseType: 'code',
+          params: {
+            scope: 'openid profile email'
+            }
+          }
+      };
+
+      var lock = new Auth0LockPasswordless('#{env.AUTH0_CLIENT_ID}', '#{env.AUTH0_DOMAIN}', opts);
+      lock.show();
     }
 
     function loginEmailLink() {
-      var lock = new Auth0LockPasswordless('#{env.AUTH0_CLIENT_ID}', '#{env.AUTH0_DOMAIN}');
-      lock.magiclink( {callbackURL: '#{env.AUTH0_CALLBACK_URL}'} );
+      var opts =  {
+        allowedConnections: ['email'],
+        passwordlessMethod: 'link',
+        auth: {
+          redirectUrl: '#{env.AUTH0_CALLBACK_URL}',
+          responseType: 'code',
+          params: {
+            scope: 'openid profile email'
```

Comment:
same

Reply:
I have mixed feelings about it, I think it's better in samples to be explicit that we are actually specifying a scope.

Reply:
:shipit: